runtime/dart_service_isolate.cc

```diff
@@ -182,10 +181,6 @@ bool DartServiceIsolate::Startup(const std::string& server_ip,
       Dart_SetField(library, Dart_NewStringFromCString("_originCheckDisabled"),
                     Dart_NewBoolean(disable_origin_check));
   SHUTDOWN_ON_ERROR(result);
-  result =
-      Dart_SetField(library, Dart_NewStringFromCString("_authCodesDisabled"),
```

Comment:
I don't think this can be deleted. This is still affecting the behavior of the vm service: https://github.com/dart-lang/sdk/blob/main/sdk/lib/_internal/vm/bin/vmservice_io.dart#L24